packages/client/hmi-client/src/components/workflow/tera-calibration.vue

```diff
@@ -1,26 +1,111 @@
+					<span class="p-buttonset">
+						<Button
+							class="p-button-secondary p-button-sm"
+							label="Input"
+							icon="pi pi-sign-in"
+							@click="calibrationView = CalibrationView.INPUT"
+							:active="calibrationView === CalibrationView.INPUT"
+						/>
+						<Button
+							class="p-button-secondary p-button-sm"
+							label="Output"
+							icon="pi pi-sign-out"
+							@click="calibrationView = CalibrationView.OUTPUT"
+							:active="calibrationView === CalibrationView.OUTPUT"
+						/>
+					</span>
+				</template>
+			</tera-asset-nav>
+		</template>
+		<template #edit-buttons>
+			<Button icon="pi pi-play" label="Run" class="p-button-sm" :disabled="isRunDisabled" />
+		</template>
+		<Accordion
+			v-if="calibrationView === CalibrationView.INPUT && modelConfig"
+			:multiple="true"
+			:active-index="[0, 1, 2, 3, 4]"
+		>
+			<AccordionTab :header="modelConfig.model.name">
+				<tera-model-diagram :model="modelConfig.model" :is-editable="false" />
+				<!-- @update-model-content="updateModelContent" -->
+			</AccordionTab>
+			<AccordionTab header="Model configuation">
+				<tera-model-configuration
+					ref="modelConfigurationRef"
+					:model="modelConfig.model"
+					:is-editable="false"
+					calibration-config
+				/>
+			</AccordionTab>
+			<AccordionTab :header="datasetName">
+				<tera-dataset-datatable :raw-content="csvAsset ?? null" />
+			</AccordionTab>
+			<AccordionTab header="Train / Test ratio"> </AccordionTab>
+			<AccordionTab header="Mapping">
+				<section class="mapping">
+					<div>
+						Select target variables from the model and the corresponding data column you want to
+						match them to.
+					</div>
+					<DataTable :value="mapping" v-model:expandedRows="expandedRows">
+						<Column expander style="width: 5rem" />
+						<Column field="modelVariable" header="Model variable">
+							<template #body="{ data, field }">
+								<Dropdown
+									class="w-full"
+									placeholder="Select a variable"
+									v-model="data[field].label"
+									:options="modelVariables"
+								/>
+							</template>
+						</Column>
+						<Column field="datasetVariable" header="Dataset variable">
+							<template #body="{ data, field }">
+								<Dropdown
+									class="w-full"
+									placeholder="Select a variable"
+									v-model="data[field].label"
+									:options="datasetVariables"
+								/>
+							</template>
+						</Column>
+						<template #expansion="slotProps">
+							<div>
+								Model: {{ slotProps.data.modelVariable }}
+								<br />
+								Dataset: {{ slotProps.data.datasetVariable }}
+								<!-- <DataTable> put above in table -->
+							</div>
+						</template>
+					</DataTable>
+					<div>
+						<Button
+							class="p-button-sm p-button-outlined"
+							icon="pi pi-plus"
+							label="Add mapping"
+							@click="addMapping"
+						/>
+					</div>
+				</section>
+			</AccordionTab>
+		</Accordion>
+		<!-- <table v-if="featureMap">
```

Comment:
If this has a use for something in the future its good, but if its test/old code commented out lets just remove it